src/main/java/ui/UI.java

```diff
@@ -585,5 +585,18 @@ public void triggerNotificationPaneAction() {
             }
         });
     }
-
+    
+    public void setTitle() {
+        String openBoard = "";
+        if (prefs.getLastOpenBoard().isPresent()) {
+            openBoard = prefs.getLastOpenBoard().get();
+        } else {
+            openBoard = "No board open";
+        }
```

Comment:
Can be replaced with
```java
String openBoard = prefs.getLastOpenBoard().orElse("No board open");
```
Also I'm not sure about `HubTurbo V1.2.3 (No board open)`. Maybe `HubTurbo V1.2.3 (none)` (terser), or hide the parentheses entirely?